### DISCUSSION TITLE
TS011F switch no longer has switch entity after update to 1.34.0 version

### DISCUSSION THREAD

**Discussion by juliuspollak** [Koenkk/zigbee2mqtt#20029] [Devices]
Hello, 
I'm on Home Assistant OS 7.1 with Deconz RaspBee

I have updated Zigbee2MQTT addon from 1.31.0 to the latest 1.34.0 to get support for some new devices. 

After the update, I have lost the switch entity/option for **some** TS011F_plug_1 (i.e. some TS011F cannot be controlled via automations and some still can, I have 5 in total and currently 2 not working). 

The State is visible from the Exposes tab of Zibbee2MQTT and working (in majority of cases). If it doesnt work (pressing either ON or OFF does nothing) I get an error message:

> Publish 'set' 'state' to '0xa4c138068bfd2b0a' failed: 'Error: Command 0xa4c138068bfd2b0a/1 genOnOff.off({}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (no response received (102))'

Using the "Reconfigure" option fails

> Failed to configure (ConfigureReporting 0xa4c138068bfd2b0a/1 haElectricalMeasurement([{"attribute":"rmsVoltage","minimumReportInterval":5,"maximumReportInterval":3600,"reportableChange":5}], {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (no response received (111)))

> Failed to configure '0xa4c138068bfd2b0a', attempt 3 (Error: ConfigureReporting 0xa4c138068bfd2b0a/1 haElectricalMeasurement([{"attribute":"rmsVoltage","minimumReportInterval":5,"maximumReportInterval":3600,"reportableChange":5}], {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (no response received (111)) at DeconzAdapter.sendZclFrameToEndpoint (/app/node_modules/zigbee-herdsman/src/adapter/deconz/adapter/deconzAdapter.ts:666:23) at runMicrotasks (<anonymous>) at runNextTicks (node:internal/process/task_queues:61:5) at processTimers (node:internal/timers:499:9) at Request.send (/app/node_modules/zigbee-herdsman/src/controller/helpers/request.ts:81:20) at Endpoint.configureReporting (/app/node_modules/zigbee-herdsman/src/controller/model/endpoint.ts:759:28) at Object.rmsVoltage (/app/node_modules/zigbee-herdsman-converters/src/lib/reporting.ts:220:5) at Object.configure (/app/node_modules/zigbee-herdsman-converters/src/devices/tuya.ts:3285:13) at Configure.configure (/app/lib/extension/configure.ts:117:13) at Configure.onMQTTMessage (/app/lib/extension/configure.ts:55:21))

Force Remove and re-join doesnt solve this problem.

Downgrade to my last backup 1.31.0 version solves this problem, but I loose support of new devices (hence the update to the latest version).

![TS011F_1](https://github.com/Koenkk/zigbee2mqtt/assets/99386240/c258094e-9b2c-445c-8f53-ed1c0e373e9a)

![TS011F_2](https://github.com/Koenkk/zigbee2mqtt/assets/99386240/a561603d-b8d5-4b5e-9400-e343742d45c4)

![TS011F_3](https://github.com/Koenkk/zigbee2mqtt/assets/99386240/a7ad3fa5-8103-4e64-b154-743796dd27e2)

**Comment by Koenkk** [comment]
Your HA version is probably too old, make sure you are running at least 2023.8.0

**Comment by juliuspollak** [comment]
Hi @Koenkk it worked. Thank you
